src/main/java/com/minecolonies/coremod/client/gui/WindowBuildBuilding.java

```diff
@@ -249,7 +249,7 @@ private void updateResources()
             }
             else
             {
-                Log.getLogger().error("WindowMinecoloniesBuildTool: Need to download schematic on a standalone client/server. This should never happen");
+                Log.getLogger().error("WindowMinecoloniesBuildTool: Need to download schematic on a standalone client/server. This should never happen", new Exception());
```

Comment:
Could we possibly create a custom exception? just because "Exception" is pretty bland XD.

Reply:
hm don't think that matters, since the log message appears above the stacktrace so you see its minecolonies and which error